doc/nrf/ug_nrf5340.rst

```diff
@@ -356,107 +356,112 @@ You must manually add a network core project to the application core project to
    You must reprogram the network core sample only when changes are made to it.
    You can modify and program the application core sample without reprogramming the network core.
 
-
 Follow these steps to build and program a multi-image build to the nRF5340 application core and network core:
 
-1. Follow the instructions in :ref:`gs_programming_ses` and open the application core sample (for example, :ref:`peripheral_lbs`) in |SES|.
-   Use ``nrf5340dk_nrf5340_cpuapp`` or ``nrf5340dk_nrf5340_cpuapp_ns`` as the build target.
-#. Build the sample as described in :ref:`gs_programming_ses`.
+1. Follow the initial steps in :ref:`gs_programming_ses` to import the application core sample project in |SES| using ``nrf5340dk_nrf5340_cpuapp`` or ``nrf5340dk_nrf5340_cpuapp_ns`` as the build target.
+   For example, select the :ref:`peripheral_lbs` sample.
+#. Build the sample as described in :ref:`gs_programming_ses` in the step about building and programming the project.
    This creates both the application core image and the network core image.
-#. Select :guilabel:`File` > :guilabel:`New Project`.
-
-    .. figure:: images/ses_nrf5340_netcore_new_project.png
-       :alt: Create New Project menu
+   When the build completes, return to this guide.
+#. Add the network core project to the application core project:
```

Comment:
Do we actually add it to an application core project or to some sort of main project ?